src/config.rs

```diff
@@ -99,7 +83,7 @@ pub(crate) fn construct_config(
             state_pruning: 1024,
             validator: true,
             name: node_name.to_owned(),
-            port: 30333,
+            port: "".to_owned(),
```

Comment:
This won't work for sure.

Reply:
You should probably don't supply any multiaddr here, but if you supply one it should be smth like `/ip4/127.0.0.1/tcp/0`.

Reply:
Why it won't work? It worked in `dev` chain? 
If you mean it won't work on actual chains, then I can put a check in the construction, and if this is an empty string, it won't use `listen_on` method on the builder.

Reply:
I mean, it shouldn't work as you just don't pass any propper value for `Multiaddr` I believe. (Or maybe `Multiaddr` here is just empty)

Reply:
yes it's empty, parse function worked fine on this empty string and I was able to farm blocks on dev chain